libr/util/file.c

```diff
@@ -242,17 +242,22 @@ R_API char *r_file_path(const char *bin) {
 				*ptr = '\0';
 				file = r_str_newf (R_JOIN_2_PATHS ("%s", "%s%s"), str, bin, extension);
 				if (r_file_exists (file)) {
+					if (strchr (file, ' ')) {
+						char *tmp = file;
+						file = r_str_newf ("\"%s\"", file);
```

Comment:
use the r_str api to escape chars instead of quoting it because that introduces other kind of vulnerabilities

Reply:
Is it possible to escape whitespace on Windows?